test/odbc/psqlodbc/psqlodbc_tests_common.h

```diff
@@ -0,0 +1,337 @@
+#ifndef PSQLODBC_TESTS_COMMON_H
+#define PSQLODBC_TESTS_COMMON_H
+
+#include <cmath>
+#include <gtest/gtest.h>
+#include <sqlext.h>
+#include <string>
+#include <time.h>
+#include <vector>
+
+#include "../src/drivers.h"
+#include "../src/odbc_handler.h"
+#include "../src/query_generator.h"
+
+using std::vector;
+using std::string;
+using std::pair;
+
+const int BUFFER_LENGTH = 8192;
+const int BUFFER_SIZE = 256;
+const int CHARSIZE = 255;
```

Comment:
CHARSIZE is used in only one function, so it probably should not be global, but just defined in that function.

Reply:
Yes, you're right. Done.